tak-web/src/main/resources/templates/pubversion/date-form.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Select Dates</title>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
Ladda ner och lägg in jquery-filerna i resources så de inte behöver hämtas externt

Reply:
Fixat. Experiment-fil borttagen.